gnomad_qc/v3/annotations/generate_freq_data.py

```diff
@@ -221,7 +227,7 @@ def main(args):
                 popmax=pop_max_expr(mt.freq, mt.freq_meta, POPS_TO_REMOVE_FOR_POPMAX),
             )
             mt = mt.annotate_globals(
-                faf_meta=faf_meta, faf_index_dict=make_faf_index_dict(faf_meta)
+                faf_meta=faf_meta, faf_index_dict=make_faf_index_dict(faf_meta, label_delimiter="-")
```

Comment:
Maybe this is a stupid question, do we need `set_female_y_metrics_to_na_expr` for faf also?

Reply:
Nope, because we run this (`set_female_y_metrics_to_na_expr`) once on the `freq` annotation before we compute the FAF annotations, so the right values go into that computation

Reply:
Ah, yes, that makes sense. thank you